### PR TITLE
standalone run

### PR DESCRIPTION
**Pull request by jasonLaster**
This makes it possible to run the launchpad by itself.

Along the way, it also simplifies the config considerably

* devtools-core has the shared babel plugins and react deps
* there's a new base developmentConfig

### CODE REVIEW

**Comment by nchevobbe** [comment]
The launchpad works well, but there's an error when selecting the Settings panel : 

screenshot at fevr 10 07-38-15

**Comment by bomsy** [comment]
Fixed the settings issue.

File: packages/devtools-launchpad/config/development.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "favicon": "assets/launchpad-favicon.png",
+  "theme": "light",
+  "dir": "ltr",
+  "logging": {
+    "client": false,
+    "firefoxProxy": false,
+    "actions": false
+  },
+  "development": {
+    "serverPort": 8000,
```
Comment (bomsy) [inline packages/devtools-launchpad/config/development.json]: Do we want to use a different port, so as not to conflict with debugger running locally?

Comment (jasonLaster) [inline packages/devtools-launchpad/config/development.json]: heh - that sounds hard to remember. anyways, we'd only want one app running and talking to firefox

Comment (bomsy) [inline packages/devtools-launchpad/config/development.json]: oh yeah ..kk

**Pull request merged**